openfecli/tests/commands/test_plan_rbfe_network.py

```diff
@@ -149,6 +152,44 @@ def test_plan_rbfe_network_cofactors(eg5_files):
 
         assert result.exit_code == 0
 
+@pytest.fixture
+def cdk8_files():
+    with resources.files("openfe.tests.data") as p:
+        if not (cdk8_dir := p.joinpath("cdk8")).exists():
+            shutil.unpack_archive(cdk8_dir.with_suffix(".zip"), p)
+        pdb_path = str(cdk8_dir.joinpath("cdk8_protein.pdb"))
+        lig_path = str(cdk8_dir.joinpath("cdk8_ligands.sdf"))
+
+        yield pdb_path, lig_path
+
+def test_plan_rbfe_network_charge_changes(cdk8_files):
```

Comment:
I wonder if it would make sense to have a test with three ligands and two edges, one with charge change and one without to test the "mixed" networks.

Reply:
Good idea I was hoping the other tests would catch any breaks in the settings but they don't seem to be checking for it.

Reply:
done in [983dce8](https://github.com/OpenFreeEnergy/openfe/pull/1053/commits/983dce809a8489d7aa7ad5a9ee76aab596601331)